examples/decoding/plot_poldrack_space_net.py

```diff
@@ -0,0 +1,45 @@
+"""
+SpaceNet on Jimura et al "mixed gambles" dataset.
+==================================================
+
+The segmenting power of SpaceNet is quiete visible here.
+"""
+# author: DOHMATOB Elvis Dopgima,
+#         GRAMFORT Alexandre
+
+
+### Load data ################################################################
+from nilearn.datasets import fetch_mixed_gambles
+data = fetch_mixed_gambles(n_subjects=16, make_Xy=True)
+X, y, mask_img = data.X, data.y, data.mask_img
+
+
+### Fit TV-L1 #################################################################
+penalty = "tv-l1"
```

Comment:
Here we don't need this intermediate variable, so let's not use it, and pass directly to SpaceNet